unified-runtime/source/adapters/level_zero/image_common.cpp

```diff
@@ -1441,10 +1441,10 @@ ur_result_t urBindlessImagesImportExternalSemaphoreExp(
         SemDesc.pNext = &FDExpDesc;
         switch (semHandleType) {
         case UR_EXP_EXTERNAL_SEMAPHORE_TYPE_OPAQUE_FD:
-          SemDesc.flags = ZE_EXTERNAL_SEMAPHORE_EXP_FLAGS_OPAQUE_FD;
+          SemDesc.flags = ZE_EXTERNAL_SEMAPHORE_EXT_FLAG_OPAQUE_FD;
           break;
         case UR_EXP_EXTERNAL_SEMAPHORE_TYPE_TIMELINE_FD:
-          SemDesc.flags = ZE_EXTERNAL_SEMAPHORE_EXP_FLAGS_TIMELINE_SEMAPHORE_FD;
```

Comment:
Why have these flags changed?

Reply:
My bad, I pushed a wrong commit. Reverted this change.